app/templates/components/forms/group/group-view.hbs

```diff
@@ -69,13 +69,12 @@
       {{/if}}
     </div>
     <div class="three wide column">
-      {{#if this.session.isAuthenticated}}
-        <button class="blue {{if this.device.isMobile 'mt-2' 'mt-4'}} ui fluid button" {{action 'toggleFollow'}}>
+        <button class="blue {{if this.device.isMobile 'mt-2' 'mt-4'}} ui fluid button" style="background-color:rgba(65, 150, 214, 0.40)" {{action 'toggleFollow'}}>
```

Comment:
We only want the alpha when the user is **logged out**. Also, the style is repeating and will be used at more places regarding groups, so we can create a class for this.

Reply:
So should I create two buttons?

Reply:
No